frontend/src/lib/i18n/init.ts

```diff
@@ -118,22 +115,6 @@ export {defaultLocale};
 // 4. Utility function exports
 /////////////////////////////////////////////////////
 
-const dynamicTranslationsAdded: Record<string, boolean> = {};
```

Comment:
We use flat key-text records for overriding the translations, so these can be added using the built-in `addTranslations` method.